Pully Penelope/Assets/Scripts/PlayerMovement.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlayerMovement : MonoBehaviour
+{
+    [SerializeField]
+    private float moveSpeed;
+
+    private Animator animator;
+    private new Rigidbody2D rigidbody;
+    private Vector3 positionChange;
+
+    private void Start()
+    {
+        rigidbody = GetComponent<Rigidbody2D>();
+        animator = GetComponent<Animator>();
+    }
+
+    private void Update()
+    {
+        positionChange = Vector3.zero;
+        positionChange.x = Input.GetAxisRaw("Horizontal");
+        positionChange.y = Input.GetAxisRaw("Vertical");
+
+        /*if (positionChange != Vector3.zero)
```

Comment:
Why is this here?

Reply:
Ah just commenting out an alternative solution in case I need it again. I'll probably delete it by the next iteration.